Formula/tarantool.rb

```diff
@@ -39,8 +36,13 @@ def install
     args << "-DENABLE_DIST=ON"
     args << "-DOPENSSL_ROOT_DIR=#{Formula["openssl@1.1"].opt_prefix}"
     args << "-DREADLINE_ROOT=#{Formula["readline"].opt_prefix}"
-    args << "-DCURL_INCLUDE_DIR=#{sdk}/usr/include"
+    args << "-DENABLE_BUNDLED_LIBCURL=OFF"
+    args << "-DCURL_INCLUDE_DIR=/usr/include"
```

Comment:
I believe this, curses and iconv include should still be 
```suggestion
    args << "-DCURL_INCLUDE_DIR=#{sdk}/usr/include"
```

Reply:
Thanks, for the suggestions, I've made them all.